components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -370,8 +370,16 @@ export default class CollectionItem extends mixins(ChainMixin, PrefixMixin) {
 }
 </script>
 
-<style>
+<style lang="scss">
+@import '@/styles/mixins';
+
 .collection__image img {
   color: transparent;
 }
+
+.tabs-container-collection {
+  .tab-content {
+    @include tabs-mobile-padding;
+  }
+}
```

Comment:
different approache would be using `class="px-0"`

Reply:
yes but since we only want this on mobile, we would also need a variable which tells us this right @roiLeo

Reply:
oh. I thought we would remove x axis padding in tab content on all view.
Your solution is interesting, why did you choose to create custom classes on each component? it's not better to create a global class like `tabs-container` and add the rule to global.scss?

Reply:
yeah I guess I could do that too, I thought instead of populating global.scss (since it already has lots of classes), I'd try to make a mixin out of it
I'll add in the global css then